conf.py

```diff
@@ -351,8 +351,11 @@ def xapian_run_example_command(ex):
         return "%s -unsafe -target:exe -out:%s.exe %s -r:XapianSharp.dll\n./%s.exe" \
             % (csc, ex, xapian_code_example_filename(ex), ex)
     elif highlight_language == 'java':
-        javac = get_tool_name('JAVAC', 'javac')
-        java = get_tool_name('JAVA', 'java')
+        java_bindings_dir = get_tool_name('JAVA_BINDINGS_DIR', 'usr/local/') # change default
```

Comment:
I was also wondering whether it is appropriate to use `get_tool_name` here. It has the functionality we want to get the value of the env variable, but our use case isn't exactly true to the naming (since we aren't actually trying to get a "tool" name). But it seems unnecessary to create another function that does the exact same thing just for naming purposes.